src/main/flight/mixer.c

```diff
@@ -755,6 +755,16 @@ static float applyThrottleLimit(float throttle)
     return throttle;
 }
 
+static float applyThrottleVbatLimit(float throttle)
+{
+    if (currentControlRateProfile->throttle_vbat_limit_percent > 0) {
```

Comment:
`if (currentControlRateProfile->throttle_vbat_limit_percent) {` makes it more explicit that '0' is a special case meaning 'off'.

Reply:
Ok. Done!